tasks/agent.py

```diff
@@ -79,7 +79,7 @@
 def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None,
           puppy=False, development=True, precompile_only=False, skip_assets=False,
           embedded_path=None, rtloader_root=None, python_home_2=None, python_home_3=None,
-          arch='x64'):
+          with_both_python=False, arch='x64'):
```

Comment:
We could use `python_home_2` and `python_home_3` instead of adding a new variable.

Reply:
I don't think so: according to the `datadog-agent` software definition, on Windows we never provide these arguments (see [here](https://github.com/DataDog/datadog-agent/pull/4013/files/b160c624b5a92ce06bc02b90f5dcae19d03322b2#diff-26a7fe78c368e14af12bd897cae6b525R63)), and on Linux/MacOS we always provide both arguments (see [here](https://github.com/DataDog/datadog-agent/pull/4013/files/b160c624b5a92ce06bc02b90f5dcae19d03322b2#diff-26a7fe78c368e14af12bd897cae6b525R68)), regardless of whether we built `rtloader` for Python 2, Python 3 or both.

Reply:
ho yeah ... That's because on Windows the Pythons location are dynamic